### DISCUSSION TITLE
Better Music visualation (fluid v2)

### DISCUSSION THREAD

**Discussion by Jak2k** [rocksdanister/lively#635] [Ideas]
**Is your feature request related to a problem? Please describe.**
No, there isn't a problem.

**Describe the solution you'd like**
I think it would be nicer if instead of many "streaks" there is one that keeps moving.

**Additional context**
![image](https://user-images.githubusercontent.com/76660118/126533487-58b2e641-cd79-4292-b83e-cf91baa0573d.png)
_So it's now._

![image](https://user-images.githubusercontent.com/76660118/126533404-f80c25b3-8ea1-4c3c-a82f-27939f3214e7.png)
_I think it would be more beautiful so._

**Comment by slst19** [comment]
In addition to this if input can be taken from Spotify music that would be even awesome.
